build.gradle

```diff
@@ -68,3 +68,19 @@ idea {
         sourceDirs += file('src/api/java')
     }
 }
+
+test {
+    workingDir = {minecraft.runDir + "/test"} // This can be set to whatever you prefer
+
+    mkdir workingDir
+    file("$workingDir/eula.txt").text = "eula=true"
+
+    file("$workingDir/server.properties").text = """
+level-type=FLAT
+generator-settings=3;minecraft:bedrock,3*minecraft:stone,52*minecraft:sandstone;2;
+online-mode=false
+level-name=integrateddynamics.test
+"""
+
+    outputs.upToDateWhen { false }
```

Comment:
Not sure what this does, is it needed?

Reply:
This causes the test task to always run, even if there have been no code changes since the last run.
I find this useful when debugging tests as I sometimes want to run them multiple times without making code changes.
However, I'll go ahead and remove it as it isn't really necessary for the CI pipeline.